src/components/tagsInput/index.js

```diff
@@ -236,24 +239,48 @@ export default class TagsInput extends BaseComponent {
     return _.get(item, 'label');
   }
 
-  renderLabel(tag, shouldMarkTag) {
+  checkTagValidate(tag, validationErrorMessage) {
+    return validationErrorMessage ? tag.invalid : false;
+  }
+
+  renderLabel(tag, shouldMarkTag, isTagNotValidate) {
```

Comment:
also here. 
you don't need the `isTagNotValidate` argument. 
you can use the `tag.invalid` instead

Reply:
changed